### DISCUSSION TITLE
JCenter shutting down

### DISCUSSION THREAD

**Discussion by csllc-one** [journeyapps/zxing-android-embedded#691] [Q&A] [answered]
Where is this dependency going to be hosted in the future?  JCenter had a brown-out today to remind everyone that it will be permanently shuttered on Feb 1.

**Comment by rkistner** [comment, marked as answer]
This has already been migrated to maven central, including versions 3.3.0, 3.5.0, 3.6.0 and 4.1.0 and any newer version. If you're on a different older version you'll need to upgrade. See https://github.com/journeyapps/zxing-android-embedded/issues/606 for details.
